component/DashBoard/component/DashboardGoalsAvatar.jsx

```diff
@@ -0,0 +1,77 @@
+import { Dropdown, Tooltip } from "antd";
+import React from "react";
+import { getFirstLetter } from "../../../helpers/truncateString";
+
+const avatarCount = 1;
+const remainingAvatarCount = avatarCount - 1;
+
+function DashboardGoalsAvatar({ activeGoalUsers }) {
+  return (
+    <div className="flex items-center">
+      {Number(activeGoalUsers.length) > 0 &&
+        activeGoalUsers.map((data, index) =>
+          index < avatarCount ? (
+            <Tooltip
+              title={data?.assignee?.first_name}
+              placement="top"
+              key={index + "users"}
+            >
+              <div
+                className={`${
+                  index === 0
+                    ? "bg-cyan-500 "
+                    : index === 1
+                    ? "bg-orange-600 -ml-2"
+                    : "bg-green-600 -ml-2"
+                } 
+            z-${index === 0 ? "0" : `${index}0`}
+           border-2 text-white flex justify-center items-center capitalize hover:cursor-pointer hover:z-50 transition-all duration-200 ease-in-out rounded-full w-7 h-7`}
+              >
+                {getFirstLetter(data?.assignee?.first_name)}
+              </div>
+            </Tooltip>
+          ) : null
+        )}
+      {Number(activeGoalUsers.length - avatarCount) > 0 && (
```

Comment:
??

Reply:
if activeGoalUsers is undefined than Number(activeGoalUsers?.length - avatarCount) will be NaN which will be false